### PR TITLE
Move template instructions and boilerplate into comments

### PR DESCRIPTION
**Pull request by petertgiles**
## 👋 Introduction

I get a bit annoyed when I have to delete a bunch of template instructional text and boilerplate when writing up an issue or PR.  I also get annoyed when said text and boilerplate ends up in posted text.  I had the idea of using comments instead.

Let me know what you think of this idea.  If anyone really doesn't like it we can close this without merging.  :grin: 

## 🕵️ Details

A few small other changes:
- Removed example blocker item from the tasklist.  Those usually aren't added in the markdown editor anyway.
- Merged designer tag into design section.  We probably don't need a whole h3 section for a user name.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/✨-feature-request.md [hunk 1]
```diff
@@ -8,44 +8,43 @@ assignees: ""
...
 
-Describe the feature that is being requested
+<!-- Describe the feature that is being requested -->
 
 ## 🕵️ Details
 
-Add any additional details that could assist with the development of the new feature.
+<!-- Add any additional details that could assist with the development of the new feature. -->
 
 ## 🎨 Design File
```
Comment (mnigh) [inline .github/ISSUE_TEMPLATE/✨-feature-request.md:17]:
```suggestion
## 🎨 Design
```

File: .github/ISSUE_TEMPLATE/🔤-copy.md [hunk 2]
```diff
@@ -8,31 +8,33 @@ assignees: ""
...
 
-Describe the update or addition to the copy that you are proposing.
+<!-- Describe the update or addition to the copy that you are proposing. -->
 
 ## 🕵️ Details
 
-Add any additional details that could assist with the implementation of the new or existing copy.
+<!-- Add any additional details that could assist with the implementation of the new or existing copy. -->
 
 ## 🎨 Design File
```
Comment (mnigh) [inline .github/ISSUE_TEMPLATE/🔤-copy.md:17]:
```suggestion
## 🎨 Design
```

File: .github/ISSUE_TEMPLATE/⚙️-new-component.md [hunk 3]
```diff
@@ -8,15 +8,15 @@ assignees: ""
 
 ## ⚙️ Summary
 
-Purpose and description.
+<!-- Purpose and description. -->
 
 ## 🎨 Design File
```
Comment (mnigh) [inline .github/ISSUE_TEMPLATE/⚙️-new-component.md:13]:
```suggestion
## 🎨 Design
```

**Review by mnigh — CHANGES_REQUESTED** [review]
I'm ok with removing the designer section and incorporating it into the design section, but the Design File heading should be renamed accordingly.

**Review by mnigh — APPROVED** [review]

**Comment by petertgiles** [comment]
This has been up a few days and no one has objected.  I'll merge.

**Pull request merged**